src/Microsoft.AspNetCore.SignalR.Core/HubEndPoint.cs

```diff
@@ -67,6 +67,12 @@ public class HubEndPoint<THub> where THub : Hub
             {
                 await _lifetimeManager.OnConnectedAsync(connectionContext);
                 await RunHubAsync(connectionContext);
+
+                await connectionContext.WriteAsync(CloseMessage.Empty());
```

Comment:
Do these sends actually get sent successfully? Because at the end of `RunHubAsync` we try to abort the connection https://github.com/aspnet/SignalR/pull/1591/files#diff-3f9cc189cbcb6b225bb8d6cb6ecdeb21R117

Reply:
My VS has been broken on this branch so I haven't tested yet.